vhdbuilder/packer/components.json

```diff
@@ -220,9 +217,8 @@
       "downloadURL": "mcr.microsoft.com/oss/tigera/operator:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.20.0",
```

Comment:
tigera v1.20.0 is still referenced in AKS RP, if you remove it too soon, AKS VHD Image Preload Checking pipeline could fail. But tigera version is switch over to v1.24.2 in AKS RP, v1.20.0 can be removed from VHD cache.

{{- else if eq .component "tigera-operator-image" -}}
{{/* The tigera-operator influences which Calico version will get used for k8s versions 1.20 and above. v1.20.0 will install Calico v3.20.0 on Linux */}}v1.20.0

Reply:
Gotcha, I originally thought that we had reverted to v1.23.1/v3.21.0. Instead I will keep v1.20.0/v3.20.0 as a backup and remove the unused.